src/parallel/LoadBalancer.h

```diff
@@ -20,12 +21,25 @@ class LoadBalancer {
 
 	/**
 	 * The rebalancing call.
-	 * Based on the current domain and the work for that domain this function determines a new 
+	 * Based on the current domain and the work for that domain this function determines a new
 	 * domain decomposition that provides a better load balancing.
 	 * This call will normally include communication and exchange of information with other processes.
 	 * @param work Arbitrary unit of work, e.g., time for the current process
-	 * @return New domain boundaries for the current process. First entry is the new boxMin, 
+	 * @return New domain boundaries for the current process. First entry is the new boxMin,
 	 * second the new boxMax.
 	 */
 	virtual std::tuple<std::array<double, 3>, std::array<double, 3>> rebalance(double work) = 0;
+
+	/**
+	 * Read Config file
+	 * @param xmlconfig
+	 */
+	virtual void readXML(XMLfileUnits& xmlconfig) = 0;
+
+	/**
+	 * Get the information in which direction this process covers the entire domain.
+	 * @return array of bools, for each dimension one value: true, iff the domain is filled only be the current process,
+	 * i.e, the process spans the entire domain.
```

Comment:
Wording unclear. So you want to indicate the case that there is only one rank / process in a certain dimension? Then maybe more like:
```suggestion
	 * Indicates if the current process / MPI rank spans the full length of a dimension.
	 * @return Array of bools, for each dimension one value: true, iff the process spans the entire domain along this dimension.
```